substrate/client/executor/polkavm/src/lib.rs

```diff
@@ -43,38 +44,18 @@ impl WasmInstance for Instance {
 		name: &str,
 		raw_data: &[u8],
 	) -> (Result<Vec<u8>, Error>, Option<AllocationStats>) {
-		let Some(method_index) = self.0.module().lookup_export(name) else {
-			return (
-				Err(format!("cannot call into the runtime: export not found: '{name}'").into()),
-				None,
-			);
-		};
-
```

Comment:
Restore this code. You can find the export with `self.0.module().exports().find(...)`.

Reply:
Yeah, again fixed based on compilation error.

Reply:
https://github.com/paritytech/polkadot-sdk/pull/6533/commits/5b375075ad76fc7a0cefc6b6212b487a6682d73c